eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/GoldenUtils.hs

```diff
@@ -34,14 +34,13 @@ import Cardano.Ledger.Binary (
  )
 
 -- ToExpr (CBOR.Term) instance
-import Cardano.Ledger.TreeDiff (diffExpr)
 import qualified Codec.CBOR.Encoding as CBOR (Encoding (..))
 import Control.Exception (throwIO)
 import Control.Monad (unless)
 import qualified Data.ByteString.Lazy as BSL (ByteString)
 import Data.String (fromString)
 import GHC.Stack
-import Test.Cardano.Ledger.Binary.TreeDiff ()
+import Test.Cardano.Ledger.Binary.TreeDiff (diffExpr)
```

Comment:
Are we putting lower bounds on dependencies in the test packages? None of the other dependencies have them.

Reply:
If the library does not depend on the package and only the test suite or testlib depends on the package then we need to put lower bounds on it. 

We don't usually put lower bounds in testlibs and tests, because those are usually the same.

Same applies in this scenario, you only need to update bound in one place:
https://github.com/input-output-hk/cardano-ledger/blob/169969bdb29a84f1b3a2c70fb42e7d5dfa39b2f2/eras/shelley/test-suite/cardano-ledger-shelley-test.cabal#L87